README.md

```diff
@@ -1 +1,87 @@
+
+![menu](./diagrams/projects.svg)
+
+#### Project Details
+
+The project details page includes the project's balance, registered addresses, transaction history, account history, and unique JSON-RPC url containing the Project ID which is used to authenticate your API calls.
+
+###### Balance
+
+This section is where you can add your credit card with FuelStop, and after which displays information about the account associated with the project. The Mainnet project would start without credit card added, and therefore you have to add one before we can fund your transactions. The Ropsten project, however, is with a testnet of Ethereum, and does not require you to add any credit card to start using. It comes with a fake credit card, and you can top it up anytime you want by clicking on the "Top Up Button" with no cost. This is where you can test your system or library with FuelStop.
+
+![Balance without credit card](./diagrams/balance-without-credit-card.svg)
+
+![FuelStop Documentation](./diagrams/balance.svg)
+
+###### Details
+
+Here you can find the name of the network the project is associated with, and your RPC URL. FuelStop's Ethereum JSON-RPC APIs require a valid project ID to be part of your URL to authenticate the calls. This is how we identify who you are behind the API requests, and it is important to keep it a secret.
+
+![Details](./diagrams/details.svg)
+
+###### Addresses
+
+Before sending your signed transactions to us via `eth_sendRawTransaction`, you need to register your address with us. If you send FuelStop a signed transaction via your RPC URL that is not included in your registered addresses, FuelStop will behave no differently than a regular Ethereum node and will broadcast it to the network. If your address has some ETH but not enough to cover the transaction's fees, FuelStop will fund your address with the exact amount needed for the transaction to go through.
+
+![Addresses](./diagrams/addresses.svg)
+
+###### Transactions
+
+The transactions table contains your transaction history. You can periodically refresh it to monitor the status of your transactions. A transaction's status will be one of the following:
+
+1. `New`: The transaction has been received and is waiting for the fund transaction to be mined.
+2. `Sent`: The transaction has been broadcasted to the network and is waiting to be mined.
+3. `Confirmed`: The transaction has been mined in a block and transaction execution is successful on-chain.
+4. `Failed`: The transaction has been mined in a block and transaction execution has failed on-chain. This is likely because you sent your transaction to a smart contract, and part of the contract logic has rejected a state transition.
```

Comment:
```suggestion
4. `Failed`: The transaction has been mined in a block and transaction execution has failed on-chain. This is likely because you sent your transaction to a smart contract, and part of the contract logic has errored out.
```

Reply:
I'm not sure if there's a better way to say it than "errored out", but we shouldn't mention a "state transition" as that's part of the inner workings of the network, and you don't need to understand that to be able to use our platform.